help/de/Help_ch_GUIMenus.xml

```diff
@@ -286,7 +286,7 @@
                   </para>
                 </entry>
               </row>
-<!-- FIXME source: check the order and item -->
+
               <row>
                 <entry namest="c2">
                   <para><guimenuitem><accel>R</accel>echnungen importieren...</guimenuitem>
```

Comment:
sieh mal in de.po:
```
#: gnucash/import-export/bi-import/dialog-bi-import-gui.c:198
msgid "Import Bills or Invoices from csv"
msgstr "Rechnungen oder Lieferantenrechnungen aus CSV-Datei importieren"

```
(Erstmal im Programm verbessern zu "Kunden- oder Lieferantenrechnungen aus CSV-Datei importieren".)
Dann sollten wir die Docs wohl mal aktualisieren. ;-)

Der Trick: Im Programm den Tooltip merken und in de.po suchen, in den Docs
- msgid -> C
-  msgstr -> de 

kopieren, Minimum fertig! Du darfst aber gern noch etwas "Soße" dran geben.